docs/appguide.rst

```diff
@@ -280,10 +280,13 @@ by swiping or using the Arrow keys to bring up the launcher and then clicking
 on your application.
 
 The application can also be registered automatically when you load the
-simulator if you add it to ``wasp/main.py``. Try adding lines 5 and 6 from
+simulator if you add it to ``wasp/boards/simulator/main.pywasp/main.py``. Try adding lines 5 and 6 from
 the above example into this file (between ``import wasp`` and
 ``wasp.system.run()``).
 
+You can use something like ``while inotifywait -e modify hello.py; do   killall python3;   nohup make sim 2>&1 & done`` to 
```

Comment:
Nice concept. However `killall python3` is pretty brutal so I think it needs to be "Kill any existing simulator processes (and any other python3 interpretters that are currently running) and (re)launch..."